indico/modules/events/registration/fields/simple.py

```diff
@@ -226,3 +231,51 @@ class EmailField(RegistrationFormFieldBase):
     @property
     def validators(self):
         return [IndicoEmail()]
+
+
+class AccommodationField(RegistrationFormFieldBase):
+    name = 'accommodation'
+    wtf_field_class = JSONField
+    versioned_data_fields = RegistrationFormBillableField.versioned_data_fields | {'accommodation_options'}
+
+    @classmethod
+    def process_field_data(cls, data, old_data=None, old_versioned_data=None):
+        unversioned_data, versioned_data = super(AccommodationField, cls).process_field_data(data, old_data,
+                                                                                             old_versioned_data)
+        items = versioned_data['accommodation_options']
```

Comment:
Yeah, could be, much shorter and we don't need to be so explicit about this name.